README.md

```diff
@@ -528,6 +571,210 @@ The [`FileLocker` class](./src/file-locker.ts) uses `beforeAll` and `afterAll` t
 
 All events are type-safe, so IDEs will prevent typos and supply strong types for the event payloads.
 
+### CLI
+
+Umzug has an optional built-in CLI helper. To use it, call the `.runAsCLI()` method from an `Umzug` instance:
+
+```js
+// migrator.js
+const { Umzug } = require('umzug')
+
+exports.umzug = new Umzug({ ... })
+
+if (require.main === module) {
+  umzug.runAsCLI()
+}
+```
+
+#### CLI Usage
+
+This script is now a runnable CLI program, complete with help text. You can run `node migrator.js --help` to see how to use it. It'll print something like
```

Comment:
```suggestion
This script is now a runnable CLI program. You can run `node migrator.js --help` to see how to use it. It will print something like:
```